main.cpp

```diff
@@ -229,7 +237,7 @@ void showShop (){
 
     RectangleShape operatesButton;
 
-    operatesButton.setSize(Vector2f (30, 30));
+    operatesButton.setSize(Vector2f (30, 30)); // Зробити створення елементів ГУІ у функції
```

Comment:
Remove space and add const like BUTTON_WIDTH, BUTTON_HEIGHT 
Vector2f(BUTTON_WIDTH, BUTTON_HEIGHT)